From e1432e06cd3bb0b5e79b2aa0b9e0aa362acb7954 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Thu, 20 May 2021 10:29:28 +0200
Subject: [PATCH] turris1x: Make USB storage support compiled-in

This should allow us to boot directly from a flash drive.
---
 target/linux/mpc85xx/config-4.14 | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/target/linux/mpc85xx/config-4.14 b/target/linux/mpc85xx/config-4.14
index 77fd10f..1ae1cc4 100644
--- a/target/linux/mpc85xx/config-4.14
+++ b/target/linux/mpc85xx/config-4.14
@@ -362,3 +362,7 @@ CONFIG_WATCHDOG_CORE=y
 # CONFIG_XPS_USB_HCD_XILINX is not set
 CONFIG_XZ_DEC_BCJ=y
 CONFIG_XZ_DEC_POWERPC=y
+CONFIG_USB_STORAGE=y
+CONFIG_USB_UAS=y
+CONFIG_SCSI=y
+CONFIG_BLK_DEV_SD=y
-- 
2.31.1

